index.js

```diff
@@ -14,51 +14,42 @@ const boxenOptions = {
     backgroundColor: "#FFFFFF"
 }
 
-
-
-const version = boxen(chalk.redBright.bold("Urltester 1.0.0"), boxenOptions);
-
-if(process.argv.length == 2 ){
-    
+const messageOne = () => {
     console.log(chalk.red("\n                Standard user manual                "));
     console.log((chalk.gray("----------------------------------------------------"))); 
     console.log(chalk.redBright("urltester filename   -"), "reports good,bad,unknown urls"); 
     console.log(chalk.redBright("urltester v |version -"), "displays tool version"); 
     console.log((chalk.redBright("----------------------------------------------------\n"))); 
-}
+}; 
 
-else {
-    if(process.argv[2] === "v" || process.argv[2] === "version") {
-        console.log(version);
-    }
-    else {      
-        const filePath = path.join(__dirname,process.argv[2]);
-        fs.readFile(filePath,'utf-8', function(err, data) {
-            if(err) {
-                console.log(chalk.red("Unsuccesful to read file"), err)
-            }
-            else {
-                const urlArr = data.match(/(http|https)(:\/\/)([\w+\-&@`~#$%^*.=/?:]+)/gi);
+const version = boxen(chalk.redBright.bold("Urltester 1.0.0"), boxenOptions);
+
+if(process.argv.length == 2 ) messageOne();
+else if(process.argv[2] === "v" || process.argv[2] === "version" || process.argv[2] === "-v") {
+    console.log(version);
+}    
+else {      
+    const filePath = path.join(__dirname,process.argv[2]);
+    fs.readFile(filePath,'utf-8', function(err, data) {
+    if(err) console.log(chalk.red("Unsuccesful to read file"), err)
+    else {
+        const urlArr = data.match(/(http|https)(:\/\/)([\w+\-&@`~#$%^*.=/?:]+)/gi);
                 
-                urlArr.forEach((url) => {
-                //network request of url
-                    fetch(url)
-                    .then(function(response) {
-                        if(response.status == 200) {
-                            console.log(chalk.green(response.status, url));
-                        }
-                        else if(response.status == 400 || response.status == 404) {
-                            console.log(chalk.red(response.status, url));
-                        }
-                        else {
-                            console.log(chalk.gray(response.status, url) );
-                        }
-                    })
-                    .catch(function(error) {
-                        console.log(chalk.redBright("Error occured "), error);  
-                    });
-                });
-            }
+        urlArr.forEach((url) => {
+            //network request of url
+            fetch(url,{method: "HEAD", timeout: 1500})
+            .then((response) => {
+            if(response.status == 200) console.log(chalk.green(response.status, url));
```

Comment:
If your "IF" just execute one call you don't need to write: 
if(true){
console.log("hello")
} 
You can just write:
if(true) console.log("hello");

Reply:
Will fix. thanks.